manifests/i/IDA/Pike/8.0.1738/IDA.Pike.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: IDA.Pike
+PackageVersion: 8.0.1738
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  Dependencies:
+    ExternalDependencies: ["Microsoft.VCRedist.2015+.x86"]

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
- InstallerLocale: en-US
  Architecture: x86
```

Reply:
I'm not sure why, but the ExternalDependencies value doesn't seem to install the `VCRedist` package.